### PR TITLE
Racks can be kicked apart

### PR DESCRIPTION
**Pull request by Konater**
I copied what probe did for glass tables so that racks can be kicked down after four hits

### CODE REVIEW

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -690,6 +691,16 @@
 				qdel(src)
 				new /obj/item/weapon/rack_parts(src.loc)
 
+/obj/structure/rack/proc/checkhealth()
+	if(health <= 0)
+		new /obj/item/weapon/rack_parts(src.loc)
```
Comment (PJB3005) [inline code/game/objects/structures/tables_racks.dm]: Src is redundant

**Comment by Konater** [comment]
So I delete the (src.loc) bit?

**Comment by PJB3005** [comment]
No only the `src.` bit.

**Comment by CptWad** [comment]
sure

**Comment by Probe1** [comment]
>copying what probe did
the blind leading the blind

**Comment by clusterfack** [comment]
And so the cycle continues

**Pull request merged**